src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/About.cshtml

```diff
@@ -4,19 +4,125 @@
 
 <div class="row content-centered">
     <div class="col-md-8">
-        <div class="row">
-            <div class="col-md-2">
-                <img src="~/images/photo_weronika_tobor.png" alt="Weronika Tobor" />
-            </div>
-            <div class="col-md-10">
-                <h2>Weronika Tobor</h2>
-                <p>Główny organizator projektu od 2017 roku oraz programista. Bloguje na <a href="http://www.programmer-girl.com">www.programmer-girl.com</a> i aktywnie udziela się we wrocławskim świecie IT. Zaangażowana m. in. w organizację Wroc# Online.</p>
-            </div>            
+        <div class="about-person">
+            <div class="row">
+                <div class="col-md-2">
+                    <img src="~/images/photo_weronika_tobor.png" alt="Weronika Tobor" width="80" />
+                </div>
+                <div class="col-md-10">
+                    <h2>Weronika Tobor</h2>
+                    <p>Główny organizator projektu od 2017 roku oraz programista. Bloguje na <a href="http://www.programmer-girl.com">www.programmer-girl.com</a> i aktywnie udziela się we wrocławskim świecie IT. Zaangażowana m. in. w organizację Wroc# Online.</p>
+                </div>
+            </div>
+            <div class="row about-social-section">
+                <div class="col-md-4"><img src="~/images/icon_www.png" alt="WWW" /><a href="http://www.programmer-girl.com">www.programmer-girl.com</a></div>
+                <div class="col-md-4"><img src="~/images/icon_fb.png" alt="Facebook" />@@programmergirlblog</div>
```

Comment:
please add link _www.facebook.com/programmergirlblog_ (here and in other profiles)